public/documents/fari-rpgs/charge-srd.md

```diff
@@ -466,9 +466,9 @@ When a player gets a `situational talent`, they need to decide on a specific sit
 
 ## Fortune Rolls
 
-When the outcome of a situation isn't certain, and the players aren't involved, the GM makes a fortune roll to see what happens. This is kind of like testing for luck.
+The fortune roll is a tool that the GM can use to disclaim decision making in a situation where the PCs aren’t directly involved in. It can also be used by the players when no other roll applies to the situation at hand. The result of the roll helps guide the outcome as to where the story goes next, but doesn't feature momentum nor consequences.
```

Comment:
Extra word

```suggestion
The fortune roll is a tool that the GM can use to disclaim decision making in a situation the PCs aren’t directly involved in. It can also be used by the players when no other roll applies to the situation at hand. The result of the roll helps guide the outcome as to where the story goes next, but doesn't feature momentum nor consequences.
```